IFC4x3/Properties/t/TrackEdgeId_3LYQ8lEh97fw__L8ZChIgF/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TrackEdgeId_3LYQ8lEh97fw__L8ZChIgF" Name="TrackEdgeId" UniqueId="d589a22f-3ab2-47a7-afbe-5488ccad2a8f" PrimaryDataType="IfcIdentifier" />
+
```

Comment:
Usual discussion regarding TrackEdgeId

Reply:
I have erased it. adb5fdd751ea322bf0649365b080a81b2a458a62

Reply:
Still here.

Reply:
Done here. a14c639b6c4248c34ffadf5422b6425f466a4277